src/modules/forms.css

```diff
@@ -49,10 +40,9 @@ textarea {
   background-color: var(--baseBackgroundColor);
   border: var(--baseBorder);
   border-radius: var(--baseBorderRadius);
-  box-shadow: var(--formBoxShadow);
   box-sizing: border-box;
   margin-bottom: var(--smallSpacing);
-  padding: calc(var(--baseSpacing) / 3);
+  padding: calc(var(--baseSpacing));
```

Comment:
We don't need the `calc` wrapping since we are not performing any operation to the var